extensions/data-plane/data-plane-spi/src/main/java/org/eclipse/dataspaceconnector/dataplane/spi/result/TransferResult.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *  Copyright (c) 2021 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+package org.eclipse.dataspaceconnector.dataplane.spi.result;
+
+import org.eclipse.dataspaceconnector.spi.response.ResponseFailure;
+import org.eclipse.dataspaceconnector.spi.response.ResponseStatus;
+import org.eclipse.dataspaceconnector.spi.result.AbstractResult;
+
+import java.util.List;
+
+/**
+ * The result of a transfer operation.
+ */
+public class TransferResult extends AbstractResult<String, ResponseFailure> {
```

Comment:
Since this doesn't really contain a `String`, `extends AbstractResult<Void, ResponseFailure>` would be better

Reply:
That may have been copy-and-paste laziness. @bscholtes1A can you change that?